setup.py

```diff
@@ -63,6 +63,7 @@ def read_reqs(path):
         'console_scripts': [
             'sopel = sopel.cli.run:main',
             'sopel-config = sopel.cli.config:main',
```

Comment:
Seeing this here reminded me that `sopel-config` (#1507) isn't included in `cli.rst` either.

Not really related to this PR, but we both agreed on IRC that this isn't worth a separate issue.